packages/vite/src/utils/versions.ts

```diff
@@ -11,5 +11,6 @@ export const happyDomVersion = '~9.20.3';
 export const edgeRuntimeVmVersion = '~3.0.2';
 
 // Coverage providers
-export const vitestCoverageC8Version = '~0.32.0';
-export const vitestCoverageIstanbulVersion = '~0.32.0';
+export const vitestCoverageC8Version = '^0.32.0';
+export const vitestCoverageV8Version = '^0.32.0';
+export const vitestCoverageIstanbulVersion = '^0.32.0';
```

Comment:
let's leave these as patch only to prevent downstream issues with other frameworks. 

ref: https://github.com/nrwl/nx/pull/18010

Reply:
Updated. Hopefully I've understood correctly